addon/sidebar-menu-manager/service.js

```diff
@@ -45,8 +45,12 @@ export default Service.extend({
 
   _menuItemsRoleFiltered: computed('_menuItems.[]', 'currentUser.roles.[]', function() {
     return this.get('_menuItems').filter(item => {
-      if (item.sidebarRole) {
-        const roles = this.get('currentUser.roles');
+      const roles = this.get('currentUser.roles');
+      if (Array.isArray(item.sidebarRole) && item.needsAllRoles) {
+        return roles && item.sidebarRole.every(role => roles.includes(role));
```

Comment:
Is there a benefit to using `roles.includes()` as opposed to `currentUserContent.hasRole()` like you did [here](https://github.com/knoxville-utilities-board/ember-nrg-ui/blob/5512bc142d6424f8dd280c36bb0a0912049de2fc/addon/components/nrg-home-cards/home-card/component.js#L22)? Just checking for consistency.